### DISCUSSION TITLE
Oracle Cloud Free Tier Server as OpenMPTCProuter

### DISCUSSION THREAD

**Discussion by usernamegeneratorFTW** [Ysurac/openmptcprouter#3380] [Ideas]
Hello!
Has anyone already tried if an "Oracle Cloud Free Tier (AMDx86 1 CPU 1GB Ram or ARM4 Cpus 24GB Ram and 200GB) server" would be sufficient/possible as an OpenMPTCProuter?

Are there any free servers that can be used as OpenMPTC routers?

**Comment by Ysurac** [comment]
Yes both are possible but use Debian image and not the default Ubuntu image.

**Comment by Network-Traditions** [comment]
We have successfully used Oracle Cloud Free Tier Server for a number of years with Debian 12 using OpenMPTCProuter VPS version 1031.  Today the VPS update script fails and we are in the process of trying to bring the system back online here #4048 .
